rbc/irtools.py

```diff
@@ -134,60 +134,35 @@ def set_env(self, env_name, env):
         return None
 
 
-class JITRemoteCPUContext(cpu.CPUContext):
-
-    @compiler_lock.global_compiler_lock
-    def init(self):
-        target_info = TargetInfo()
-        self.address_size = target_info.bits
-        self.is32bit = (self.address_size == 32)
-        self._internal_codegen = JITRemoteCPUCodegen("numba.exec")
-
-    def get_executable(self, library, fndesc, env):
-        return None
-
-    def post_lowering(self, mod, library):
-        pass
-
+class JITRemoteTypingContext(typing.Context):
+    def load_additional_registries(self):
+        from rbc.omnisci_backend import omnisci_mathtyping as mathtyping
 
-# ---------------------------------------------------------------------------
-# GPU Context classes
+        self.install_registry(mathtyping.registry)
```

Comment:
This approach needs to be generalized. Namely, users (including the rbc project) may provide `omnisci_mathtyping`-similar modules that should be installed similarly what is in the above, BUT without changing the irtools.py

Reply:
The same behavior can be achieve using different approaches already available in RBC/Numba: For instance:
- External
- extending.overload_*